manifests/rhoai/shared/odhdashboardconfig/odhdashboardconfig.yaml

```diff
@@ -28,6 +28,7 @@ spec:
     disableModelMesh: false
     disableDistributedWorkloads: false
     disableModelRegistry: false
+    disableModelRegistrySecureDBEA: true
```

Comment:
Remove this field for development features.

Reply:
[Read more here](https://redhat-internal.slack.com/archives/C05SMJ09DD2/p1733754298033679?thread_ts=1730922294.428399&cid=C05SMJ09DD2)